device_coupler/config/daq_config.yaml

```diff
@@ -0,0 +1,27 @@
+# DAQ FOT configuration.
+
+include: ${DAQ_LIB}/config/system/default.yaml
+
+host_tests: ${DAQ_LIB}/config/modules/all.conf
+
+base_conf: local/base_config.json
+finish_hook: bin/dump_network
+test_config: resources/runtime_configs/long_wait
+site_path: inst/test_site
+schema_path: schemas/udmi
+monitor_scan_sec: 20
+
+switch_setup:
+  lo_port: 6653
+  varz_port: 5678
+  data_intf: trunk0
+
+run_trigger:
+  vlan_start: 200
+  vlan_end: 205
+  auto_session: True
+
+device_reporting:
+  server_port: 50051
+
+docker_bridge_ip: 172.17.0.1
```

Comment:
this shouldn't be "docker" -- what's it used for? (Not where does it come from.)  Is this the address that the gRPC client tries to connect to?  Also, likely should be under some subsection not at the top-level.  Is this used by the device-coupler inside of its docker container, right?  Now that I think more about it, it might be better as an ENV variable just passed in when the coupler container is started... would that be easy to do?

Reply:
I think that's should be doable, but it necessitates the integration of DAQ and device_coupler.

Reply:
I moved docker_bridge_ip into run_trigger instead.

Reply:
You missed the part about the name (docker_ prefix)

Reply:
Welp. Renamed to grpc_target_ip